src/main/java/gregtech/api/metatileentity/multiblock/MultiblockControllerBase.java

```diff
@@ -343,9 +343,9 @@ public void checkStructurePattern() {
                     abilityPart.registerAbilities(abilityInstancesList);
                 }
             }
-            parts.forEach(part -> part.addToMultiBlock(this));
             this.multiblockParts.addAll(parts);
             this.multiblockAbilities.putAll(abilities);
+            parts.forEach(part -> part.addToMultiBlock(this));
```

Comment:
What is this change accomplishing? Could probably use a comment stating why this changed in the code.

Reply:
Calling `addToMultiblock()` after all parts are added to the internal parts lists, so that you can use that method to do things depending on what the other parts in the structure are